e2e/version/version.go

```diff
@@ -74,9 +76,22 @@ func (c *ctx) testEqualVersion(t *testing.T) {
 					t.Fatalf("FAIL: singularity version command and singularity --version give a non-matching version result")
 				}
 			} else {
-				tmpVersion = outputVersion
+				tmpVersion = outputVer
 			}
-		})
+		}
+
+		e2e.RunSingularity(
+			t,
+			tt.name,
+			e2e.WithCommand(tt.args[0]),
```

Comment:
```suggestion
			e2e.WithArgs(tt.args),
```

Reply:
on this one, if you see the definition of tests variable, the first arg is a command ( "version" ) and the second one is an arg flag ("--version") passing them as parameter in the WithCommand is working by now for both cases since those are in a loop, but if there's a better way to do this do lmk, thanks! @cclerget

Reply:
@devprincess Yep I saw, `WithCommand` purpose is to prepend the command as part of singularity arguments, so to take singularity commands like `shell`, `version`, `instance start` ... not really suitable for hyphens arguments, that's why I suggested to use `WithArgs` because you will have the same result as with `WithCommand`, so it should work

Reply:
cool! I'll do the appropriate changes then, thanks @cclerget !